django/db/backends/postgresql/psycopg_any.py

```diff
@@ -0,0 +1,39 @@
+import ipaddress
+
+try:
+    from psycopg import IsolationLevel, errors, sql
+    from psycopg.types.range import Range
+
+    Inet = ipaddress.ip_network
```

Comment:
There's an issue with CockroachDB related to this because psycopg's `NetworkDumper` that [serializes](https://github.com/psycopg/psycopg/blob/008a6efed97f6f892247f911426f0717c24412a1/psycopg/psycopg/types/net.py#L193-L194) `IPv4Network` and `IPv6Network` (the result of parsing an address as seen in the example below) uses the [cidr type](https://github.com/psycopg/psycopg/blob/008a6efed97f6f892247f911426f0717c24412a1/psycopg/psycopg/types/net.py#L61) which [isn't implemented in CockroachDB](https://github.com/cockroachdb/cockroach/issues/18846). Thus there are failures when trying to save Django's `GenericIPAddressField`: `django.db.utils.OperationalError: unknown oid type: 650` (650 is the oid for cidr.)

```python
>>> import ipaddress
>>> ipaddress.ip_network('192.0.2.100')
IPv4Network('192.0.2.100/32')
```
Offhand, I'm not sure if we could implement a workaround django-cockroachdb or if CockroachDB may consider adding https://github.com/cockroachdb/cockroach/issues/18846 to the list of tasks for psycopg3 support. \cc @rafiss

Reply:
@apollo13, any reason not to use `ipaddress.ip_address` here instead? As far as I tested, this resolves the above issue on CockroachDB and passes on PostgreSQL also.

Reply:
No specific reason, will have to double check. This is/was still part of the initial PR from Daniele IIRC.

Reply:
This comment is still pending.

Reply:
Ah sorry I forgot about this one. Technically using `cidr` is the correct type here. Would this be hard for the cockroach db backend to override? I mean in the end the builtin backend is for postgresql…

Reply:
Actually you are right, `ip_address` would be the better option here since we only support adresses and not networks.